### PR TITLE
i18n(es): Update markdown-content.md

### PR DESCRIPTION
**Pull request by agustinmulet**
#### What kind of changes does this PR include?

- New or updated content

#### Description

- Updates `markdown-content.md` in spanish

- [x] ⚠️ We should update `layouts.md` since there's an inexistent link in line 324 of this PR ⚠️

### CODE REVIEW

**Comment by dreyfus92** [comment]
Buen trabajo Agus, LGTM ✅

**Comment by delucis** [comment]
Thanks for reviewing @dreyfus92! We should be able to merge this once #2177 is reviewed & merged.

File: src/pages/es/guides/markdown-content.md [hunk 1]
```diff
@@ -31,245 +27,174 @@ title: Hola mundo
...
+  <p>{frontmatter.description}<p>
+  <slot /> <!-- Aquí se inyecta el contenido Markdown -->
+   <!-- ... -->
+</html>
+```
 
-Este es mi artículo de blog en progreso.
-
-No se creará ninguna página para esta publicación.
+📚 Aprende más acerca de [Plantillas de Markdown](/es/core-concepts/layouts/#plantillas-de-markdown).
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/markdown-content.md:132]:
```suggestion
📚 Aprende más acerca de [Plantillas de Markdown](/es/core-concepts/layouts/#plantillas-de-markdownmdx).
```

File: src/pages/es/guides/markdown-content.md [hunk 2]
```diff
@@ -320,68 +314,46 @@ const posts = await Astro.glob<Frontmatter>('../pages/post/*.md');
...
-```
-
-#### `rawContent()`
-
-Una función que devuelve el contenido sin procesar del archivo Markdown (excluyendo el bloque frontmatter) como un string. Esto es útil cuando, por ejemplo, se calculan los "minutos leídos". Este ejemplo usa el popular paquete [reading-time](https://www.npmjs.com/package/reading-time):
-
-:::tip
-¡Si planeas utilizar `rawContent` para calcular valores como "tiempo de lectura", te sugerimos usar un complemento de remark o rehype para inyectar frontmatter! Puedes ver nuestro [ejemplo de cálculo de tiempo de lectura](#ejemplo-calcular-tiempo-de-lectura) para más información.
+:::note[¿Usando una plantilla de Astro?]
+Puedes ver las [propiedades exportadas a un componente de plantilla de Astro](/es/core-concepts/layouts/#props-de-plantilla-de-markdown) cuando usas el [layout en el Frontmatter](#layout-en-el-frontmatter) especial de Astro
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/markdown-content.md:324]:
```suggestion
Puedes ver las [propiedades exportadas a un componente de plantilla de Astro](/es/core-concepts/layouts/#props-de-plantillas-markdown) cuando usas el [layout en el Frontmatter](#layout-en-el-frontmatter) especial de Astro
```

**Review by kevinzunigacuellar — COMMENTED** [review]
gracias por la PR @agustinmulet, deje un suggestion para los links rotos

**Pull request merged**